content/lang/checklists/checklistloader.es.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/*
+------------------
+Language: Spanish
```

Comment:
We're starting a practice in which we list who translated the file and when. Could you please add these metadata?
Translator: Google Translate (unless it's you!)
Date Translated: YYYY-MM-DD